Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -103,14 +103,15 @@ class WebTreeView
 
   private static OnKeyDown (event: KeyboardEvent, treeView: HTMLElement): void
   {
-    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('li[role=treeitem]'));
+    const treeNodes = Array.from (treeView.querySelectorAll<HTMLElement> ('a[role=treeitem]'));
+    const parentTreeNodeListItems = Array.from(treeView.querySelectorAll<HTMLElement>('li'));
```

Comment:
I don't like that we do two separate queries and assume that their elements relate to each other in the correct order. Since `li` is always a parent of `a[role=treeitem]` we should transform the first list into the second using `.closest("li")`.

Maybe it makes sense to put them into an object to group them so that we don't have to maintain two separate but related arrays. We would need to use the `.findIndex` instead of `indexOf` below instead.

Reply:
I dont think we need to add them to an object. They are const, so we dont really have to maintain them. It might be overkill having them both to begin with instead of just querying for the required `htmlElement`